shell/list/harvesterhci.io.management.cluster.vue

```diff
@@ -79,6 +80,21 @@ export default {
     typeDisplay() {
       return this.t(`typeLabel."${ HCI.CLUSTER }"`, { count: this.row?.length || 0 });
     },
+  },
+
+  methods: {
+    async goToCluster(row) {
+      try {
+        setTimeout(() => {
+          // Don't show loading indicator for quickly fetched plugins
+          this.navigating = row.id;
+        }, 1000);
+        await row.goToCluster();
+      } catch {
+        // The error handling is carried out within goToCluster
+        this.navigating = false;
```

Comment:
The harvester cluster's `goToCluster` is `.catch`'ing all plugin loading so this isn't hit when that fails -  when I try to nav to a cluster for which plugin loading will fail, I get the error growl as expected but `navigating` remains true (infinite load spinner & un-clickable cluster links). The PR works well for plugin loading that eventually succeeds though!

Reply:
Great catch! I've now fully tested and fixed the error case